pkg/externalresource/control.go

```diff
@@ -79,22 +79,6 @@ func (c *K8sControl) List(ctx context.Context, ns string, labelSelector string)
 	return result, nil
 }
 
-func (c *K8sControl) Validate(ctx context.Context, name, ns string, externalInfo *ExternalResource) error {
-	k8sclient, err := c.ClientProvider(c.Cfg)
-	if err != nil {
-		return fmt.Errorf("error creating external CRD client: %s", err.Error())
-	}
-
-	externalResourceCRD := translate(name, externalInfo)
-
-	_, err = k8sclient.ExternalResources(ns).Create(ctx, externalResourceCRD, metav1.CreateOptions{DryRun: []string{metav1.DryRunAll}})
```

Comment:
Why this fixes the issue if the create is done with a `DryRun` which means that the changes won't be persisted in Kubernetes?

Reply:
even having that option, the CRD was being created. I will create a follow up issue to try to introduce a validation again properly

Reply:
Cool, thanks!!